packages/cli/lib/prompts/sandboxesPrompt.js

```diff
@@ -17,6 +17,18 @@ const mapSandboxAccountChoices = portals =>
       };
     });
 
+const mapNonSandboxAccountChoices = portals =>
+  portals
+    .filter(
+      p => p.sandboxAccountType === null || p.sandboxAccountType === undefined
+    )
+    .map(p => {
+      return {
+        name: `${p.name} (${p.portalId})`,
+        value: p.name || p.portalId,
+      };
+    });
+
```

Comment:
Edge case bug fix for when a config entry does not have a parentAccountId. It prompts you for a parent account, but previously it would show only sandbox entries which is incorrect.